src/components/Buttons/ButtonsWelcome/ButtonsWelcome.tsx

```diff
@@ -0,0 +1,12 @@
+import { ButtonProps } from '../../../types/types';
+import './ButtonsWelcome.scss';
+
+const ButtonsWelcome = ({ text, cb }: ButtonProps) => {
```

Comment:
Plural? I see here a single button. The same problem with name as with "ButtonForm"